webapp/first_snap/content/c/package.yaml

```diff
@@ -0,0 +1,125 @@
+    - code: |
+        name: dosbox
+    - warning: |
+        <span class="p-notification__status">Snap names are globally unique.</span> You should change any instance of <code>dosbox</code> to <code>test-dosbox-{name}</code> (where <code>{name}</code> is your name) before continuing to avoid conflicting with the official DOSBox snap.
+    - text: |
+        Versions carry no semantic meaning in snaps. Use text that best identifies that release, it won't be used to compare against the preceding or following one.
+    - code: |
+        version: 0.74-svn
+    - text: The summary can not exceed 79 characters. You can use a pipe ‘|’ in the description key to declare a multi-line description.
+    - code: |
+        description: |
+  security:
+    - text: The next section describes the level of confinement applied to your app.
+    - code: |
+        confinement: devmode
+    - text: Snaps are containerised to ensure more predictable application behaviour and greater security.
+    - text: It’s best to start a snap with the confinement in warning mode, rather than strictly applied. This is indicated through the <code>devmode</code> keyword.
+    - text: Once an app is working well in devmode, you can review confinement violations, add appropriate interfaces, and switch to strict confinement.
+  base:
+    - text: In general, snaps cannot see the root filesystem on end user systems. This prevents conflict with other applications and increases security. However, applications still need some location to act as the root filesystem. They would also benefit from common libraries (e.g. libc) being in this root filesystem rather than bundled into each application.
+    - text: The base keyword specifies a special kind of snap that provides a minimal set of libraries common to most applications. It will be mounted as the root filesystem for your application.
+    - code: |
+        base: core18
+    - text: The <code>core18</code> base is recommended.
+  parts:
+    - text: |
+        Parts define what sources are needed to assemble your app. Parts can be anything: programs, libraries, or other needed assets. In this case we have one: the DOSBox source release tarball. In other cases these can point to local directories, remote git repositories or other revision control systems.
+    - text: |
+        Before building the part, the dependencies listed as <code>build-packages</code> are installed.
+    - code: |
+        parts:
+          dosbox:
+            plugin: autotools
+            source-type: tar
+            source: http://source.dosbox.com/dosboxsvn.tgz
+            build-packages:
+              - g++
+              - make
+              - libsdl1.2-dev
+              - libpng-dev
+              - libsdl-net1.2-dev
+              - libsdl-sound1.2-dev
+              - libasound2-dev
+    - text: |
+        The <code>stage-packages</code> section specifies the list of libraries required by the DOSBox snap at runtime.
```

Comment:
Something like:
>The <code>stage-packages</code> section specifies the list of libraries required by the DOSBox snap at runtime. These will be included in the snap, ensuring that every system can run dosbox regardless of what libraries are already installed.

This also looks disconnected to the rest of the parts definition. I'd consider showing the whole parts section once with the explanatory text beneath it, like we do on the other guides.